src/message.ts

```diff
@@ -613,6 +625,45 @@ export class Message implements Sayable {
   // }
 
   /**
+   * @param sendTo UserId/RoomId/Room/Contact
+   */
+  public forward(room: Room): Promise<boolean>
+  public forward(contact: Contact): Promise<boolean>
+  public forward(id: string): Promise<boolean>
+  public forward(sendTo: string|Room|Contact): Promise<boolean> {
+    const m = <MsgRawObj>this.rawObj
+    const newMsg = <MsgRawObj>{}
+    let id = ''
+    if (sendTo instanceof Room || sendTo instanceof Contact) {
+      id = sendTo.id
+    } else if (typeof sendTo === 'string') {
+      id = sendTo
+    }
+    newMsg.ToUserName = id
+    newMsg.FromUserName = config.puppetInstance().userId || ''
+    newMsg.isTranspond = true
+    newMsg.MsgIdBeforeTranspond = m.MsgIdBeforeTranspond || m.MsgId
+    newMsg.MMSourceMsgId = m.MsgId
+
+    // The following parameters need to be overridden after calling createMessage()
+
+    // If you want to forward the file, would like to skip the duplicate upload, sendByLocal must be false.
+    // But need to pay attention to file.size> 25Mb, due to the server policy restrictions, need to re-upload
```

Comment:
By using fiddler to intercept data, testing found files larger than 25mb in webwx can not use mediaId for multiplexing, the server will return processing failure.
I think this problem needs to be shelved and later resolved. Need to refactor the upload file flow.

```Json
"BaseResponse": {
"Ret": 1,
"ErrMsg": ""
}
```